audio/libdca/Portfile

```diff
@@ -18,11 +17,13 @@ livecheck.type      regex
 livecheck.regex     {Current release is <a.*>libdca-(.*).tar.bz2</a>}
 livecheck.url       ${homepage}
 
-master_sites        https://download.videolan.org/pub/videolan/libdca/${version}/
+master_sites        https://code.videolan.org/videolan/libdca/-/archive/${version}/
```

Comment:
Why switch from the official download tarball to an autogenerated repository archive that we have to run autoreconf on?

Reply:
Sorry, somehow it was not found for me. Let me fix it.

Reply:
We still need autoconf, but I will revert the master_site change.

Reply:
> Sorry, somehow it was not found for me.

Maybe you were looking at the livecheck or web site, which still says 0.0.6 is the latest even though 0.0.7 was released four years ago. I'll try to report that to them.

> We still need autoconf

Oh yes I see. I just assumed the 0.0.7 release tarball would include a pre-generated configure script like the 0.0.6 tarball did. Whenever you make a port use autoreconf, it's a good idea to add a comment explaining why.

Reply:
> web site […] still says 0.0.6 is the latest […]. I'll try to report that to them.

Well either I have an account there and can't remember or reset the password, or someone has taken my customary username. Either way I'm not reporting anything there at the moment but if you want to their issue tracker is here:

https://code.videolan.org/videolan/libdca/-/issues/

Reply:
I won’t bother with their tracker at the moment either, but I have added a comment to the portfile re autoconf.